lib/jormungandr/src/Cardano/Wallet/Jormungandr/Api.hs

```diff
@@ -95,7 +95,8 @@ type Api =
 
 -- | Retrieve a block by its id.
 type GetBlock
-    = "v0"
+    =  "api"
```

Comment:
This `"api"` prefix is actually now hard-coded in Jörmungandr (used to be in the config). So I've hard-coded it here too as part of the expected path. Why? Because we were getting it from the `localhostBaseUrl` helper we had where we artificially added it. Now however, the URL is determined from the `--config` given to Jörmungandr which does not contain the api prefix.